src/Bicep.LangServer/Telemetry/TelemetryConstants.cs

```diff
@@ -9,17 +9,17 @@ public static class TelemetryConstants
 
         public static class EventNames
         {
-            public const string LinterStateOnBicepFileOpen = nameof(LinterStateOnBicepFileOpen);
-
             public const string NestedResourceDeclarationSnippetInsertion = nameof(NestedResourceDeclarationSnippetInsertion);
```

Comment:
probably makes sense to lower-case all these, too, and maybe add a comment as to why lowercase

Reply:
Added comment here:
https://github.com/Azure/bicep/commit/78761642f2387d0d47e71750cbb421c045f2f732

Leaving the names as is for readability and consistency.  Created https://github.com/Azure/bicep/issues/5374 to review and update remaining event names. We can revisit this then.